keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/Health.kt

```diff
@@ -27,8 +27,21 @@ data class Health(
   val warmup: Duration = Duration.ofSeconds(600),
   val healthCheckType: HealthCheckType = HealthCheckType.EC2,
   val enabledMetrics: Set<Metric> = emptySet(),
-  val terminationPolicies: Set<TerminationPolicy> = setOf(TerminationPolicy.OldestInstance)
+  val terminationPolicies: Set<TerminationPolicy> = setOf(TerminationPolicy.Default)
 ) {
   fun toClusterHealthSpec() =
     ClusterSpec.HealthSpec(cooldown, warmup, healthCheckType, enabledMetrics, terminationPolicies)
+
+  fun toClusterHealthSpecWithoutDefaults() =
+    ClusterSpec.HealthSpec(
+      if (cooldown == defaults.cooldown) { null } else { cooldown },
+      if (warmup == defaults.warmup) { null } else { warmup },
+      if (healthCheckType == defaults.healthCheckType) { null } else { healthCheckType },
+      if (enabledMetrics == defaults.enabledMetrics) { null } else { enabledMetrics },
+      if (terminationPolicies == defaults.terminationPolicies) { null } else { terminationPolicies }
+      )
```

Comment:
super minor nit but the indent looks off here, I'm surprised spotless didn't reformat

Reply:
Hmm this same thing happened on another PR by Gal - I wonder if something is funky with spotless?

Reply:
Yeah, I was thinking the same thing... I had it catch a couple of indentation issues before, so I dunno...

Reply:
BTW, if I hit Code > Reformat code in IntelliJ, will that use the google format plugin we're using, and would that match the rules in spotless?

Reply:
You have to set that up. If you run `./gradlew sA` on the command line it'll reformat it. This might help you set up in [intellij](https://stackoverflow.com/questions/42979700/how-to-configure-google-java-code-formatter-in-intellij-idea-17)

Reply:
It was already setup/enabled for me, but it doesn't look like it reformats the code on-the-fly. I had to manually click on Code > Reformat code, FYI.